cmd/minikube/cmd/start_test.go

```diff
@@ -46,7 +46,7 @@ func (r *RecordingRunner) Run(command string, args ...string) error {
 var testDir string
 var testRunner *RecordingRunner
 var testMachineConfig = cluster.MachineConfig{
-	OpenShiftVersion: "v1.3.1",
+	OpenShiftVersion: tests.OPENSHIFT_VERSION,
```

Comment:
-1, you cannot just upgrade the version. Now you broke it for 1.3.1. Not saying that it makes much sense to use an older version of OpenShift, but we don't have to break it intentionally either.